suite-native/app/android/app/proguard-rules.pro

```diff
@@ -11,6 +11,7 @@
 
 -keep class com.facebook.hermes.unicode.** { *; }
 -keep class com.facebook.jni.** { *; }
+-keep public class com.horcrux.svg.** {*;}
```

Comment:
just info: I think we have proguard disabled and we miss most of rules for other packages here, but you can keep this here because I want to try enable it in future